seismicpro/src/refractor_velocity/utils.py

```diff
@@ -36,3 +39,78 @@ def postprocess_params(params):
     if is_1d:
         return params[0]
     return params
+
+def calc_df_to_dump(rv):
```

Comment:
I'd change the signature to
```suggestion
def calc_df_to_dump(*refractor_velocities):
```
and construct a single dataframe from all of them at once. This will simplify code a bit an will be a bit more clear